packages/server/src/scripts/arpaTreasuryReport.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+const tracer = require('dd-trace').init(); // eslint-disable-line no-unused-vars
+const { ReceiveMessageCommand, DeleteMessageCommand } = require('@aws-sdk/client-sqs');
+const { log } = require('../lib/logging');
+const { getSQSClient } = require('../lib/gost-aws');
+const { processSQSMessageRequest } = require('../arpa_reporter/services/generate-arpa-report');
+
+async function main() {
+    let shutDownRequested = false;
+    const requestShutdown = (signal) => {
+        log.warn({ signal }, 'Shutdown signal received. Requesting shutdown...');
+        shutDownRequested = true;
+    };
+    process.on('SIGTERM', requestShutdown);
+    process.on('SIGINT', requestShutdown);
+
+    const queueUrl = process.env.TREASURY_REPORT_TASK_QUEUE_URL;
```

Comment:
By default `sqs_consumer_task` configures an environment variable of `TASK_QUEUE_URL`. 

It's possible to override this in Terraform by configuring the new module like this:
```terraform
queue_url_environment_variable_name = "TREASURY_REPORT_TASK_QUEUE_URL"
```

but I would recommend leaving it as-is and just updating the env var here:
```suggestion
    const queueUrl = process.env.TASK_QUEUE_URL;
```